querying.livemd

```diff
@@ -82,16 +94,16 @@ defmodule Tutorial.Profile do
   end
 
   attributes do
-    uuid_primary_key :id
-    attribute :name, :string
+    uuid_primary_key(:id)
```

Comment:
need to make sure to remove any parens added.

Reply:
Fixed.